packages/forms/src/UIForm/fieldsets/Array/ArrayItem.component.js

```diff
@@ -0,0 +1,66 @@
+import React, { PropTypes } from 'react';
+import { Icon } from 'react-talend-components';
+
+import theme from './ArrayItem.scss';
+
+export default function ArrayItem(props) {
+	const {
+		children,
+		hasMoveDown,
+		hasMoveUp,
+		id,
+		index,
+		onRemove,
+		onReorder,
+		value,
+	} = props;
+	return (
+		<div className={theme['tf-array-item']} >
```

Comment:
Could you add a CSS class in addition to the CSS module one ? 

`<div className={'tf-array-item', theme['tf-array-item']} >`